contracts/interfaces/CloberOrderBook.sol

```diff
@@ -276,6 +286,10 @@ interface CloberOrderBook {
      */
     function bestPriceIndex(bool isBid) external view returns (uint16);
 
+    function blockTradeLogIndex() external view returns (uint16);
```

Comment:
@JhChoy, I noticed the function name `getBlockLatestTradeLogIndex` seems a bit long and might be a bit unclear to someone who is not familiar with the codebase. Would you consider shortening it to `getLatestTradeLogIndex` to make it more concise and easier to understand?

Reply:
It seems that the function returns zero when there is no `tradeLogIndex`. Is this the intended behavior? If so, could you please document this in a comment within the function code? Otherwise, we may need to consider a different way to handle this scenario.

Reply:
- The name of this function is not `getBlockLatestTradeLogIndex`. If you intend to add a `get` prefix, I think it doesn't have to do b/c we designated the view functions of the properties as the same name as the name of the properties like `quoteUnit`, `makerFee`, etc.
- Returning `-1` when the array is empty looks great to me.

Reply:
Oh, but we can't return `-1` because the type is unsigned int.